urllib3/connectionpool.py

```diff
@@ -40,9 +40,14 @@
 from .util.response import assert_header_parsing
 from .util.retry import Retry
 from .util.timeout import Timeout
-from .util.url import get_host, Url
+from .util.url import get_host, Url, NORMALIZABLE_SCHEMES
 from .util.queue import LifoQueue
 
+
+if six.PY2:
```

Comment:
This looks like an automatic rebase failure on GitHubs part. Could you rebase your changes against the latest master?

Reply:
It was problem on my side. I did a wrong rebase yesterday.